examples/dynamics/controlled_integrator.ipynb

```diff
@@ -232,7 +233,8 @@
       "plt.plot(t, ref, 'k--', label='Exact')\n",
       "plt.plot(t, sim.data[A_probe][:,0], label='A (value)')\n",
       "plt.plot(t, sim.data[A_probe][:,1], label='A (control)')\n",
-      "plt.ylim([-1.1, 1.1])\n",
+      "plt.xlim(right=t[-1])\n",
```

Comment:
According to the [docs](http://matplotlib.org/api/pyplot_api.html#matplotlib.pyplot.xlim), this argument should be called `xmax` not `right`.